node.js

```diff
@@ -155,7 +155,33 @@ export default class Node extends Tool {
      * @returns {Promise<[string | null, boolean | null]>} Resolved node version
      */
     async getNodeVersion(desiredVersion) {
-        // If we're given a version, it's the one we want
+        // Handle major version numbers like "20"
+        if (desiredVersion && /^\d+$/.test(desiredVersion)) {
```

Comment:
Maybe instead of chaning the code here you could do it inside https://github.com/open-turo/action-setup-tools/blob/main/node.js#L110 where we already have some more logic to pick the right node version?